src/variables.jl

```diff
@@ -283,6 +283,41 @@ struct VariableNotOwned{V<:AbstractVariableRef} <: Exception
     variable::V
 end
 
+function Base.showerror(io::IO, err::VariableNotOwned)
+    return print(
+        io,
+        """
+        $err: the variable $(err.variable) cannot be used in this model because
+        it belongs to a different model.
+
+        ## Explanation
+
+        Each variable belongs to a particular model. You cannot use a variable
+        inside a model to which it does not belong. For example:
+
+        ```julia
+        model = Model()
+        @variable(model, x >= 0)
+        model2 = Model()
+        @objective(model2, Min, x)  # Errors because x belongs to `model`
+        ```
+
+        The most common cause of this error is working with a main problem and a
```

Comment:
I don't know if that's "the most common". Most of the time, I'm just using a notebook or executing line by line in VS code and I get this error because I forgot to execute one part.

Reply:
Changed to "One common"